input/resources/au-core-requester.xml

```diff
@@ -3924,7 +3950,7 @@
 <name value="status"/>
 <definition value="http://hl7.org/fhir/SearchParameter/Observation-status"/>
 <type value="token"/>
-<documentation value="The requester **SHALL** provide at least a code value and **MAY** provide both the system and code values.&#xA;&#xA;The responder **SHALL** support both."/>
+<documentation value="The requester **SHALL** provide at least a code value and **MAY** provide both the system and code values.&#xA;&#xA;The responder **SHALL** support both.&#xa;&#xa;The requester **SHOULD** support `multipleOr`.&#xa;&#xa;The responder **SHOULD** support `multipleOr`."/>
```

Comment:
The expectation for 'multipleOr' should be SHALL

Reply:
fixed